core/templates/components/common-layout-directives/navigation-bars/side-navigation-bar.component.ts

```diff
@@ -32,7 +32,8 @@ export class SideNavigationBarComponent {
   // https://github.com/oppia/oppia/wiki/Guide-on-defining-types#ts-7-1
    @Input() display!: boolean;
    currentUrl!: string;
-   classroomSubmenuIsShown: boolean = false;
+   classroomSubmenuIsShownLEARN: boolean = true;
```

Comment:
maybe use `learnClassroomSubmenuIsShown` and `involvedClassroomSubmenuIsShown`

Reply:
done!